RouteOption.h

```diff
@@ -19,6 +19,7 @@ typedef NS_ENUM(NSInteger, RouteType) {
 @property (assign, nonatomic) RouteType type;
 @property (strong, nonatomic) NSDictionary *routeJson;
 @property (strong, nonatomic) NSArray *waypoints;
+@property int numOmitted;
```

Comment:
Darn you Xcode D:

Also, responding to the top-level comment (which I can't reply to): all omitted stops are sorted to the end since the pathfinding ignores them, so there isn't a version with "middle" stops grayed out. I've tested the system w/ points that are too expensive (which get correctly filtered out), but I haven't implemented waypoint omission with the distance constraint – shouldn't be difficult, but was planning to do next week after I get offline mode MVP out.

Reply:
Yeah, the waypoints at the end makes sense. 👍